jruby-9.4/force-resolv-gem-dependency.patch

```diff
@@ -0,0 +1,13 @@
+--- a/lib/pom.rb
++++ b/lib/pom.rb
+@@ -86,8 +86,8 @@
+     # Will be solved with readline
+     # ['readline-ext', '0.1.4'],
+     ['reline', '0.4.2'],
+-    # https://github.com/ruby/resolv/issues/19
+-    # ['resolv', '0.2.1'],
++    # Newer resolv gem has build issues with jruby
```

Comment:
Can you add a link to the "issues" which could either be this PR or an open issue describing the problem and what we expect the fix to be.